src/openrct2-ui/windows/Footpath.cpp

```diff
@@ -1242,3 +1242,100 @@ static bool footpath_select_default()
         return true;
     }
 }
+
+void window_footpath_keyboard_shortcut_turn_left()
+{
+    rct_window* w = window_find_by_class(WC_FOOTPATH);
+    if (w == nullptr || WidgetIsDisabled(w, WIDX_DIRECTION_NW) || WidgetIsDisabled(w, WIDX_DIRECTION_NE)
+        || WidgetIsDisabled(w, WIDX_DIRECTION_SW) || WidgetIsDisabled(w, WIDX_DIRECTION_SE)
+        || gFootpathConstructionMode != 2)
+    {
+        return;
+    }
+    int32_t currentRotation = get_current_rotation();
+    int32_t turnedRotation = gFootpathConstructDirection - currentRotation + (currentRotation % 2 == 1 ? 1 : -1);
+    window_footpath_mousedown_direction(turnedRotation);
+}
+
+void window_footpath_keyboard_shortcut_turn_right()
+{
+    rct_window* w = window_find_by_class(WC_FOOTPATH);
+    if (w == nullptr || WidgetIsDisabled(w, WIDX_DIRECTION_NW) || WidgetIsDisabled(w, WIDX_DIRECTION_NE)
+        || WidgetIsDisabled(w, WIDX_DIRECTION_SW) || WidgetIsDisabled(w, WIDX_DIRECTION_SE)
+        || gFootpathConstructionMode != 2)
```

Comment:
```suggestion
        || WidgetIsDisabled(w, WIDX_DIRECTION_SW) || WidgetIsDisabled(w, WIDX_DIRECTION_SE) || gFootpathConstructionMode != 2)
```
To please the code formatting check